### ISSUE TITLE
How to record visitor ip?

### ISSUE THREAD

**Issue by tonyhauuk** [ipkn/crow#350]
Hello huys.

 I'm using crow on my website, now I want to record the IP who access my website, such like access_log, Apache. 

Thanks!!!

**Comment by AKJ7** [comment]
How are you using it on your website?
I know C++, but don't know how to run C++ on the web, except for CGI.
Can you please tell me how you did it? 

Thanks.

**Comment by iamybj** [comment]
c++ web framework are all just a program , use ningx to serve them.

**Comment by patrick--** [comment]
I don't think this is included currently - but there are forks that have added that functionality https://github.com/mrozigor/crow/commit/741ba25abfe296e3f572fe3f23e0319bf1441c78
